sample-apps/nodejs-java/k8s/app.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  # Unique key of the Job instance
+  name: nodeshowcase-app
+spec:
+  selector:
+    matchLabels:
+      app: nodeshowcase-app
+  template:
+    metadata:
+      name: nodeshowcase-app
+      labels:
+        app: nodeshowcase-app
+    spec:
+      containers:
+        - name: nodeshowcase-app
+          image: %REGISTRY_LOCATION%-docker.pkg.dev/%GCLOUD_PROJECT%/%CONTAINER_REGISTRY%/nodejs-sample:latest
+          command: ['node', 'app.js']
+          env:
+          - name: "SERVICE_NAME"
+            value: "http://javashowcase-service/"
+          resources:
+            requests:
+              memory: "256Mi"
+              cpu: "250m"
+            limits:
+              memory: "256Mi"
+              cpu: "250m"
+      # Do not restart containers after they exit
```

Comment:
Not super familiar with Kubernetes, but does this comment seem a little contradicting to `restartPolicy: Always` ?

Reply:
yup, good catch, bad copypasta. updated in other places too